test/e2e/nsx_networkinfo_test.go

```diff
@@ -181,7 +183,7 @@ func assureNetworkInfo(t *testing.T, ns, networkInfoName string) (res *v1alpha1.
 	defer deadlineCancel()
 	err := wait.PollUntilContextTimeout(deadlineCtx, 1*time.Second, defaultTimeout, false, func(ctx context.Context) (done bool, err error) {
 		res, err = testData.crdClientset.CrdV1alpha1().NetworkInfos(ns).Get(context.Background(), networkInfoName, v1.GetOptions{})
-		t.Logf("Get NetworkInfos: %v, Namespace: %s, Name: %s, error: %v", res, ns, networkInfoName, err)
+		log.V(2).Info("Get NetworkInfos: %v, Namespace: %s, Name: %s", res, ns, networkInfoName)
```

Comment:
This is not commented? It looks like all the logs in this files below this are not in key value format?

Reply:
ok, miss this file. fixed.